package.json

```diff
@@ -101,6 +101,37 @@
                 ]
             }
         ],
+        "languageModelTools": [
+            {
+                "displayName": "Azure Resources: Get Azure Activity Log",
+                "icon": "$(azure)",
+                "inputSchema": {
+                    "type": "object",
+                    "properties": {
+                        "treeId": {
+                            "type": "string",
+                            "description": "The ID of the selected activity tree item (optional). If not specified, all items in the activity log will be focused on with equal importance."
+                        },
+                        "promptType": {
+                            "type": "string",
+                            "description": "Specifies the type of prompt to supply, such as whether to explain an item or fix an issue in the activity log (optional).",
+                            "enum": [
+                                "fix",
+                                "explain"
+                            ]
+                        }
+                    }
+                },
+                "modelDescription": "Gets the Azure activity log",
+                "name": "azureResources_getAzureActivityLog",
+                "canBeReferencedInPrompt": true,
+                "toolReferenceName": "azureActivityLog",
+                "tags": [
+                    "azure",
+                    "vscode_editing"
```

Comment:
We should verify if this line is still needed. Previously, Agent mode would only call tools with this tag.

Reply:
Oh okay good to know, I will test it without the tag